Sources/SmileID/Classes/DocumentVerification/Model/OrchestratedDocumentVerificationViewModel.swift

```diff
@@ -323,6 +324,8 @@ class IOrchestratedDocumentVerificationViewModel<T, U: JobResult>: ObservableObj
                 self.step = stepToRetry
             }
             if case .processing = stepToRetry {
+                networkRetries += 1
```

Comment:
We also need to add it for the 2215 retries here - https://github.com/smileidentity/ios/blob/0a8712441702a1845956a8136ce73362d12bc856/Sources/SmileID/Classes/DocumentVerification/Model/OrchestratedDocumentVerificationViewModel.swift#L225